python/requirements.txt

```diff
@@ -18,7 +18,7 @@ pyyaml
 aiosignal
 frozenlist
 requests
-virtualenv>=20.0.24
+virtualenv>=20.0.24, != 20.21.1
```

Comment:
Is it a virtualenv bug? Should we do `<20.21.1`?

Reply:
The reasoning is that if it's not their issue then the next virtualenv will also break things.

Reply:
i've no idea. let's first see if this deflakes it.